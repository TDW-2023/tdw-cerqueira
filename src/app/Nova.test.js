import React from 'react';
import renderer from 'react-test-renderer';
import Nova from './page';

test('snapshot para  o Componente de teste', () => {
  const component = renderer.create(<Nova />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
