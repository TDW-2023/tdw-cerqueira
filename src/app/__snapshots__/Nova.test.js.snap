// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot para  o Componente de teste 1`] = `
<main
  className="main"
>
  <div
    className="description"
  >
    <p>
      Get alright by editing 
      <code
        className="code"
      >
        src/app/page.js
      </code>
    </p>
    <div>
      <a
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        By
         
        <img
          alt="Vercel Logo"
          className="vercelLogo"
          data-nimg="1"
          decoding="async"
          fetchpriority="high"
          height={24}
          onError={[Function]}
          onLoad={[Function]}
          src="/vercel.svg"
          style={
            {
              "color": "transparent",
            }
          }
          width={100}
        />
      </a>
    </div>
  </div>
  <div
    className="center"
  >
    <img
      alt="Next.js Logo"
      className="logo"
      data-nimg="1"
      decoding="async"
      fetchpriority="high"
      height={37}
      onError={[Function]}
      onLoad={[Function]}
      src="/next.svg"
      style={
        {
          "color": "transparent",
        }
      }
      width={180}
    />
  </div>
  <div
    className="grid"
  >
    <a
      href="/nova"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      <h2>
        Nova Página 
        <span>
          -&gt;
        </span>
      </h2>
      <p
        role="beleza"
      >
        O Cerqueira é lindo!
      </p>
    </a>
    <a
      className="card"
      href="https://nextjs.org/learn?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
      rel="noopener noreferrer"
      target="_blank"
    >
      <h2>
        Learn 
        <span>
          -&gt;
        </span>
      </h2>
      <p>
        Learn about Next.js in an interactive course with quizzes!
      </p>
    </a>
    <a
      className="card"
      href="https://vercel.com/templates?framework=next.js&utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
      rel="noopener noreferrer"
      target="_blank"
    >
      <h2>
        Templates 
        <span>
          -&gt;
        </span>
      </h2>
      <p>
        Explore the Next.js 13 playground.
      </p>
    </a>
    <a
      className="card"
      href="https://vercel.com/new?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
      rel="noopener noreferrer"
      target="_blank"
    >
      <h2>
        Deploy 
        <span>
          -&gt;
        </span>
      </h2>
      <p>
        Instantly deploy your Next.js site to a shareable URL with Vercel.
      </p>
    </a>
  </div>
</main>
`;
